docs/create-webhook.yaml

```diff
@@ -137,6 +137,20 @@ spec:
       fi
       cat <<EOF | kubectl apply -f -
       apiVersion: tekton.dev/v1alpha1
+      kind: TriggerBinding
```

Comment:
The task should only conditionally create an `EventListener` not a `TriggerBinding` or `TriggerTemplate` since although empty stubs do pass validation, this task wouldn't be as useful. Can you remove this and move them to where this is being tested?

Reply:
@vtereso 
There is already e2e test in `test/eventlistener_test.go`
The reason I add `TriggerBinding` and `TriggerTemplate` here is the new added validation broke the process here.

I'm not sure if I got your point, do you mean I should add more meaningful stuff to `TriggerBinding` and `TriggerTemplate` rather than leave them empty?
I read the `doc/create-webhook.yaml`, the purpose is create a [webhook](https://github.com/tektoncd/experimental/tree/master/webhooks-extension) to listen on `github` and create `EventListener` (cannot work) to received the event.

Reply:
hey @vincent-pli is the problem that now that we have validation, in order to create a event listener, we need to add at least one triggerbinding and one triggertemplate? 

In that case, instead of creating empty TriggerBindings or TriggerTemplates, we should expect them as inputs somehow? Maybe a reference to an already existing binding/template? 

Or we should have a default template that does something useful?

Reply:
I guess i need some suggestion from the author 😄 , 
@akihikokuroda 
Could you give some suggestion? thanks

Reply:
I believe that the trigger binding and the trigger template should not be defined here for the reuseability.  They should be predefined and referenced from here.

Reply:
Thanks everyone, I have removed the `TriggerBinding` and `TriggerTemplate` from `Task` and reuse the definitions in `/examples`.